www/%username/account/index.html.spt

```diff
@@ -202,6 +202,15 @@ emails = participant.get_emails()
             {% endif %}
         </p>
 
+        <h2>{{ _("Supporter Identity") }}</h2>
+        <p>
+            <label class="anonymous-supporters">
+                <input type="checkbox"
+                    {% if not participant.anonymous_supporters %}checked="true"{% endif %} />
+                    {{ _("Allow my supporters to divulge themselves.") }}
```

Comment:
I don't feel like this is very descriptive. If you want to keep it we should at least have hover state help text.